runtime/syntax/rmd.vim

```diff
@@ -225,6 +259,10 @@ unlet s:save_pandoc_lngs
 let &cpo = s:cpo_save
 unlet s:cpo_save
 
+if has("patch-7.4.1142")
+  syntax iskeyword clear
```

Comment:
>   syntax iskeyword clear

1) I don't think we need this. 
2) we can leave out the test for patch 7.4.1142, by now Vim should always have it.